rest/adminapitest/admin_api_test.go

```diff
@@ -981,137 +981,6 @@ func TestResyncUsingDCPStreamReset(t *testing.T) {
 
 }
 
-func TestResyncForNamedCollection(t *testing.T) {
```

Comment:
Is this test already covered by a different one? Can't see this being moved anywhere in this PR.

Reply:
This test never worked but I actually removed it in https://github.com/couchbase/sync_gateway/commit/0b60724d38365885c471a3bd30a82a4cf25fa223

because it made the skip:

```
	_, ok := (rt.GetDatabase().ResyncManager.Process).(*db.ResyncManager)
	if !ok {
		t.Skip("This test only works when ResyncManager is used")
	}
```

and this coverage for this has always been in `TestResyncUsingDCPStreamForNamedCollection`.